.github/workflows/pr-sonar.yml

```diff
@@ -0,0 +1,25 @@
+name: Sonarqube scanner
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: self-hosted
```

Reply:
ahhh might be ok, since it's public

Reply:
afair self-hosted runners are not avail (yet?) for our public repos though.